dev/wlp-jakartaee-transform/rules/jakarta-versions.properties

```diff
@@ -93,3 +93,8 @@ org.jboss.weld.serialization.spi=[4.0,5)
 org.jboss.weld.transaction.spi=[4.0,5)
 org.jboss.weld.util.collections=[4.0,5)
 org.jboss.weld=[4.0,5)
+
+jakarta.security.auth.message=[2.0,3)
+jakarta.security.auth.message.callback=[2.0,3)
+jakarta.security.auth.message.config=[2.0,3)
+jakarta.security.auth.message.module=[2.0,3)
```

Comment:
Should these be added as well to be 2.0,3?

 jakarta.security.enterprise
 jakarta.security.enterprise.authentication.mechanism.http
 jakarta.security.enterprise.credential
 jakarta.security.enterprise.identitystore

Reply:
I think I just added these lines for packages I had issues with. I didn't do them for every package I created. I will need to look back at the reasoning.

Reply:
I modified the bnd.bnd for the io.openliberty.jakarta.security-2.0 bundle to use versoin 2.0.0 and then updated the file above to reflect that change.